pkg/controller/populators/import-populator.go

```diff
@@ -248,8 +248,16 @@ func (r *ImportPopulatorReconciler) updateImportProgress(podPhase string, pvc, p
 	if err != nil {
 		return err
 	}
+	if importPod == nil {
+		_, ok := pvc.Annotations[cc.AnnPopulatorProgress]
+		// Initialize the progress once PVC Prime is bound
+		if !ok && cc.IsBound(pvcPrime) {
+			cc.AddAnnotation(pvc, cc.AnnPopulatorProgress, "N/A")
```

Comment:
This is a nasty patch to force a reconcile in the DV controller when the PVC' gets bound disguised as an annotation to initialize progress. It should fix the error in the tests, and it matches the behavior in the DV controller. After spending hours on this I think this is the best-looking solution.